resources/Schema/Components/Indicator.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
```

Comment:
I probably would have gone with some other name than "Indicator". What is it indicating? If you want you could rename it to something more easily understandable "PlayerTeamIndicator" or whatever. Up to you tho